plugins/jira-dashboard-backend/src/service/router.ts

```diff
@@ -215,6 +217,42 @@ export async function createRouter(
     },
   );
 
+  router.get('/dashboards/userIssues', async (request, response) => {
+    const { token } = await auth.getPluginRequestToken({
+      onBehalfOf: await auth.getOwnServiceCredentials(),
+      targetPluginId: 'catalog',
+    });
+
+    const credentials = await httpAuth.credentials(request);
+    const info = await userInfo.getUserInfo(credentials);
```

Comment:
In the other Jira components, we allow the guest user. However, by adding this line, the guest user recivies the following exception:

500: Error

Message
Only user credentials are supported

Stack Trace
Error: Only user credentials are supported
    at DefaultUserInfoService.getUserInfo (/home/fridaja/projects/backstage-plugins/node_modules/@backstage/backend-defaults/src/entrypoints/userInfo/DefaultUserInfoService.ts:44:13)
    at <anonymous> (/home/fridaja/projects/backstage-plugins/plugins/jira-dashboard-backend/src/service/router.ts:227:33)

Request
GET /dashboards/userIssues?maxResults=15

Is there any particular reason for not retreiving the user entity like how it's done [here](https://github.com/AxisCommunications/backstage-plugins/blob/main/plugins/jira-dashboard-backend/src/service/router.ts#L157)? Then the guest user can be logged in.

Reply:
Code updated. It should return empty list for guest user now. I would like to avoid catching exception in this case - it can be handled without it.